### DISCUSSION TITLE
Already provided all the proof needed to verify my school

### DISCUSSION THREAD

**Discussion by Blink1000** [community/community#65365] [GitHub Education]
### Hi everyone,

Provided all the proof needed to verify that I am a student of my school, and I can't re-apply again because it shows that the image is edited. Wtf is this. I am literally a student. Please verify me I just want to use github co-pilot is this really that hard?

**Comment by Blink1000** [comment]
"Your application cannot be reviewed until you fix the following:
Your uploaded proof has already been used previously."

bruh what do you want me to provide. I already provided my ID, and I can't send it again

**Comment by arafajs** [comment]
- To resolve this issue, you will need to provide a new document that proves your student status. This document could be a different ID, a transcript, or a class schedule.

- You can also try contacting GitHub Education support again and explaining the situation. They may be able to help you resolve the issue.

- Here is the link to the GitHub Education support contact form: https://education.github.com/contact
